src/TweetOverlay.tsx

```diff
@@ -230,6 +230,7 @@ const TwitterContent: React.FunctionComponent<
     let in_reply_to_status_id = tweetId;
     let success = true;
     const links: string[] = [];
+    const regexTags = /(#?\[\[([^\]]*)\]\])/g;
```

Comment:
Solving the nested tags problem actually can't be done with regexes - you would need XRegExp.

A workaround could be to just replace all `#[[` and `]]` with empties, but I don't think that's necessary as that also hits some false positives